src/blob_gc.h

```diff
@@ -46,8 +56,10 @@ class BlobGC {
   bool trigger_next() { return trigger_next_; }
 
  private:
-  std::vector<BlobFileMeta*> inputs_;
```

Comment:
All variable "inputs" need rename in other files, such as "blob_gc_job_test.cc"

Reply:
fixed